src/components/EditCoursePage/EditCoursePage.test.jsx

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import {
```

Comment:
Nit: Put these on one line. I think we typically only go to multiple lines when eslint complains about line length